content/docs/wrp/simple-messages.md

```diff
@@ -96,9 +104,10 @@ great at sending metrics, publishing a report, sending an SOS.
 Name | Description
 -----|--------------
 msg_type | The message type for the simple event.  (**SHALL** be 4.)
-source | The device_id name of the device originating the request or response.
-dest | The device_id name of the target device of the request or response.
-content_type | (optional) The media type of the payload.
+source | The device_id name of the device originating the event (i.e. mac:112233445566/config)
+dest | A value of the form: `event:{event-type}`. The main use case is the ability to listeners to register only for events they care about.
```

Comment:
```suggestion
dest | An event locator of the form: `event:{event identifier}/{ignored}`. The main use case is for listeners to register only for events they care about.
```
I'm not sure we should really have the form here, since it is also on this page (which is where I got this from):
https://xmidt.io/docs/wrp/basics/#event-locators